core/src/main/java/greencity/exception/handler/CustomExceptionHandler.java

```diff
@@ -177,6 +178,19 @@ public final ResponseEntity<Object> handleConstraintViolationException(Constrain
         return ResponseEntity.status(HttpStatus.BAD_REQUEST).body(exceptionResponse);
     }
 
+    /**
+     * Handles IllegalArgumentException.
+     *
+     * @return Error message "Bad Request".
```

Comment:
This method does not return `message "Bad Request"`
it returns `HttpStatus.BAD_REQUEST` and the message will depend on the input params